src/views/2MainMap/MainMap.vue

```diff
@@ -137,15 +137,7 @@
             <text class="text-year">2019</text>
             <text class="text-year">2020</text>
 
-<<<<<<< HEAD
-             <g id="states">
-=======
             <g id="states">
-              <path
```

Comment:
deleted this path b/c it was displaying really weirdly - and it was just the San Juan islands in WA